.travis.yml

```diff
@@ -23,6 +24,7 @@ matrix:
     - rvm: jruby-9.0.1.0
```

Comment:
The `allow_failures` section now contains a non-included item. Remove the line `- rvm: jruby-9.0.1.0`